### ISSUE TITLE
False positive when a `customHOC` receives a component wrapped in `forwardRef`

### ISSUE THREAD

**Issue by tylerlaprade** [ArnaudBarre/eslint-plugin-react-refresh#79]
```tsx
export default withPeriodSpecificGridWrapper(
  forwardRef(SiteGrid),
  PeriodGridBlobType.SITE_COST_MATRIX,
);
```

This is in a file that declares the React component `SiteGrid` (and doesn't export it). I never use `SiteGrid` standalone - the `forwardRef` and `withPeriodSpecificGridWrapper` are effectively part of the component itself. It's my understanding this should be allowed given my config:

```json
		"react-refresh/only-export-components": [
			"error",
			{
				"allowConstantExport": true,
				"customHOCs": ["withPeriodSpecificGridWrapper"]
			}
		],
```

Also, I'm not seeing any Vite errors when editing this file like I do with the files with actual problematic exports.

**Comment by ArnaudBarre** [comment]
Is there a reason you don't apply forwardRef directly on `SiteGrid `?
The current lint logic for customHooks expect the first argument to be an identifier.
I can update the logic to make this works but this need some refacto because the handling is currently quite hard to follow

**Comment by ArnaudBarre** [comment]
Fixed in [0.4.20](https://github.com/ArnaudBarre/eslint-plugin-react-refresh/releases) by https://github.com/ArnaudBarre/eslint-plugin-react-refresh/commit/4ddbb1fff579c227b2036d4b0762afb2fe51399e
